### PR TITLE
Fixes #548 and #568. Added fossasia project tab to nav-bar

### LINKED ISSUE #548: Add a link on navbar going to fossasia projects
### Expected results
Another link was added when scrolled goes fossasia project section
### Actual results
There is no current tab to do so
### Steps to reproduce the issue
Update head.html
## Would you like to work on this
Yes i am working on this

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Fixes issue #548 and #568 
Updated head.html, index.html

### CODE REVIEW

**Comment by arjunpandey2003** [comment]
https://arjunpandey2003.github.io/gci17.fossasiauodce.org/.

**Review by nzec — COMMENTED** [review]
Why did you delete the `/welcome` button?

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Please add back the Welcome nav button

**Comment by arjunpandey2003** [comment]
@akshatnitd @duttaditya18 since there is already a link to the official fossasia website in the events dropdown, i removed the welcome button and linked the fossasia logo to the gci fossasia home page(Replaced the welcome button). Plus if there is that button my fossasia projects will come in and the (eng)dropdown doesnt come in.

**Review by akshatnitd — APPROVED** [review]
LGTM :) Wait for one more approval!

File: _includes/header.html
Comment (the-codingninja) [inline _includes/header.html:54]: This li tag is not closed

**Review by the-codingninja — CHANGES_REQUESTED** [review]
Check my comment

**Comment by arjunpandey2003** [comment]
@ms10398 done

**Comment by arjunpandey2003** [comment]
squashing

**Review by isuruAb — APPROVED** [review]
LGTM

File: _includes/header.html
Comment (isuruAb) [reply _includes/header.html]: please check the line 55

**Review by gabru-md — APPROVED** [review]
Merging

**Comment by gabru-md** [comment]
Great!

**Pull request merged**
